pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "cognite-sdk"
 
-version = "7.20.1"
+version = "7.20.2"
```

Comment:
Not necessary?

Reply:
I'd like the docs to be published for us to pick it up for AI consumption (we use the deployed SDK as documentation source).

Reply:
Will be updated without version bump